### DISCUSSION TITLE
Brazilian Portuguese language leader

### DISCUSSION THREAD

**Discussion by fellipute** [OHF-Voice/intents#458] [Language Leaders]
Hi, I would like to volunteer to help with the Brazilian language support.

**Comment by gianheck** [comment]
I would also like to contribute with Brazilian Portuguese support.

**Comment by guieiras-dev** [comment]
I can help contributing with Brazilian Portuguese support too.

**Comment by synesthesiam** [comment]
Merged PR from @Darigaf for `pt-br`

Is anyone here interested in being a language leader for Brazilian Portuguese?

**Reply by Darigaf** [reply]
I'm interested, if it is possible.

**Reply by dclobato** [reply]
> Is anyone here interested in being a language leader for Brazilian Portuguese?

I'd like to be

**Reply by synesthesiam** [reply]
@dclobato Can you submit some small PR's for `pt-BR`?

**Reply by dclobato** [reply]
> @dclobato Can you submit some small PR's for `pt-BR`?

done :)

**Reply by dclobato** [reply]
> @dclobato Can you submit some small PR's for `pt-BR`?

@synesthesiam I opened the PR https://github.com/home-assistant/intents/pull/1028

**Reply by dclobato** [reply]
@synesthesiam my bad PR #1028.  Now, I have the correct environment and tested before opening https://github.com/home-assistant/intents/pull/1052

**Reply by mhentschke** [reply]
I'd like to candidate as well. Just opened my second PR [#1178](https://github.com/home-assistant/intents/pull/1178) and the first one was [#1168](https://github.com/home-assistant/intents/pull/1168)

**Reply by luyzfernando08** [reply]
I'm interested

**Comment by mendoncart** [comment]
I want to collaborate, but I don't get the whole syntax for the intents... If there's someone to help me polish the intents, I'd be glad to help!

**Comment by synesthesiam** [comment]
Well, I really dropped the ball on this one :grimacing: 
Sorry to everyone! Is there anyone who still wants to volunteer?

**Reply by mhentschke** [reply]
I'm still interested in being language leader 😉

**Reply by luyzfernando08** [reply]
I'm interested

**Reply by mendoncart** [reply]
Me too.

**Comment by synesthesiam** [comment]
I see several merged PR's from @mhentschke already and one outstanding a large one from @luyzfernando08, so I will add them as  language leaders. Please look at the PR from @luyzfernando08 and coordinate to get it merged.

@mendoncart please submit some additional small PR's and we'll consider adding you as a language leader :slightly_smiling_face:

**Reply by mendoncart** [reply]
I don't need to be a "leader" I just want to contribute with whatever I can. The only problem is that I didn't understood 100% how this files work. Is there any documentation or video explaining how it works? I can make some PRs after understanding exactly how to write the intent.

**Reply by synesthesiam** [reply]
Here is some documentation on the template syntax: https://developers.home-assistant.io/docs/voice/intent-recognition/template-sentence-syntax
